src/main/resources/trinkets.mixins.json

```diff
@@ -3,23 +3,24 @@
   "package": "dev.emi.trinkets.mixin",
   "compatibilityLevel": "JAVA_17",
   "mixins": [
-    "accessor.ScreenHandlerAccessor",
+    "ExperienceOrbEntityMixin",
     "LivingEntityMixin",
     "PlayerManagerMixin",
     "PlayerScreenHandlerMixin",
     "ServerPlayNetworkHandlerMixin",
-    "ExperienceOrbEntityMixin"
+    "accessor.ScreenHandlerAccessor",
```

Comment:
Don't adjust the order of the mixins here, subpackages (`accessor`) go first.

Reply:
Going to blame the ide here, but going to fix that